mlflow/h2o.py

```diff
@@ -60,6 +69,21 @@ def log_model(h2o_model, artifact_path, **kwargs):
     :param h2o_model: H2O model to be saved.
     :param artifact_path: Run-relative artifact path.
     :param kwargs: kwargs to pass to ``h2o.save_model`` method.
+
+    >>> import mlflow
+    >>> import mlflow.h2o
+    >>> import h2o
+    >>> from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+    >>> #Partial example code used from h2o documentation <http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/early_stopping.html>
```

Comment:
nit: line is too long.

Reply:
The code snippet for load_pyfunc() has been removed in the last commit? Do you still see it.
I'll break the long line into two lines and commit.